resources/assets/coffee/react/_components/comment-show-more.coffee

```diff
@@ -30,7 +30,14 @@ class @CommentShowMore extends React.PureComponent
       loading: false
 
 
+  componentWillUnmount: =>
+    @xhr?.abort()
+
+
   render: =>
+    return null if @props.comments.length >= @props.total
+    return null unless (@props.moreComments[@props.parent?.id ? null] ? true)
```

Comment:
probably needs check for `@props.moreComments` or set `defaultProps` else it might explode when viewing permalinked comments with deep nested replies.

We might want to consider setting `defaultProps` on the other components too

Reply:
time to start using `propTypes` :trollface: